go_bot/modules/utils/chat_status/chat_status.go

```diff
@@ -156,3 +166,18 @@ func contains(s []string, e string) bool {
 	}
 	return false
 }
+
+func admincache(chat *ext.Chat) {
+	x, err := chat.GetAdministrators()
```

Comment:
Same variable naming scheme applies here

Reply:
Ok, i'll change it